zuul.d/secrets.yaml

```diff
@@ -292,3 +292,19 @@
           lrXbndwKw0rh4xJHXPfAODUpzTTjc9xBHOfQEkzjLnSiAhKG33SXZfXul950vqa23FnDT
           yy1feMpQaIKbsbC/lmhRBgdzOoegEag0oni0qV2/kh3Je4VgDM2a6055bV03Vepru8jLt
           DoLJcAQmjC6S7nPj+d6uPOsc/etFFctaGciXKSPykIs8NkY/8Yi55nOP/aRN/s=
+
+
+- secret:
+    name: ansible_core_ci
```

Comment:
this secret is unused, it a job needs to be created in project-config, and it assigned to it.